presto-verifier/src/main/java/com/facebook/presto/verifier/rewrite/FunctionCallRewriter.java

```diff
@@ -97,17 +98,10 @@ public static Multimap<String, FunctionCallSubstitute> validateAndConstructFunct
             if (specs.size() != 2) {
                 throw new IllegalArgumentException(String.format("Original function call and substitute must both be specified, %s.", substitute));
             }
-            Expression originalExpression = parseOriginalFunctionCall(specs.get(0));
+            Expression originalExpression = parseOriginalExpression(specs.get(0));
```

Comment:
Why rename this method? I'm guessing because cast is an expression but not a function call, but then why is this class FunctionCallRewriter and not ExpressionRewriter? and SUPPORTED_ORIGINAL_FUNCTIONS instead of SUPPORTED_ORIGINAL_EXPRESSIONS?

Reply:
Renaming SUPPORTED_ORIGINAL_FUNCTIONS to SUPPORTED_ORIGINAL_FUNCTIONS.

Yes, renaming parseOriginalFunctionCall to parseOriginalExpression to be more accurate in code, as we will support rewriting more than function calls. Calling the class FunctionCallRewriter so that the user can tell what this class generally does intuitively.

Reply:
can we name the class expression rewriter if it's expanded beyond function calls? And similarly rename other things from FunctionCall to Expression (e.g. functionCallSubstitute -> ExpressionSubstitute, makeFunctionCallSubstituteMapKey -> makeExpressionSubstituteKey).

Reply:
Hi @rschlussel , I thought it over, and still feel it a bit strange if we call it Expression.* but only support two types of expression. We can call it FunctionCall.* and take Cast as an extension.

However, I'm not quite sure, as both of you point it out. What do you think? I can certainly refactor.

Reply:
I would prefer to rename. It's okay that currently there are only 2 kinds of expressions it supports rewriting.